### ISSUE TITLE
Interface elements need color contrast boost on hover and focus events

### ISSUE THREAD

**Issue by bkawula** [scholarsportal/accessibility#3]
Reported by Mark Weiler.  See: https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html E.g., .nav-item:focus, .nav-item:hover, button:focus, button:hover { background-color: gold !important; color: black !important; } .form-control:focus, option:focus{ background-color: gold !important; color: black; }

**Comment by bkawula** [comment]
Fixed!
